### ISSUE TITLE
Fatal exception in dryrun with trailing stoploss on exchange

### ISSUE THREAD

**Issue by il-katta** [freqtrade/freqtrade#2194]
* Operating system: docker on linux
  * Python Version: 3.7.3
  * CCXT version: 1.18.992
  * Branch: Master
  * Last Commit ID: 0d9d23a888d1189a6c9b6e6d453d0aee08183724
  
## Step 3: Describe the problem:

got this error message when I try to run in dryrun mode

```
freqtrade - ERROR - Fatal exception!
 Traceback (most recent call last):
   File "/freqtrade/freqtrade/main.py", line 49, in main
     worker.run()
   File "/freqtrade/freqtrade/worker.py", line 72, in run
     state = self._worker(old_state=state)
   File "/freqtrade/freqtrade/worker.py", line 110, in _worker
     self._throttle(func=self._process, min_secs=throttle_secs)
   File "/freqtrade/freqtrade/worker.py", line 123, in _throttle
     result = func(*args, **kwargs)
   File "/freqtrade/freqtrade/worker.py", line 134, in _process
     state_changed = self.freqtrade.process()
   File "/freqtrade/freqtrade/freqtradebot.py", line 135, in process
     state_changed |= self.process_maybe_execute_sell(trade)
   File "/freqtrade/freqtrade/freqtradebot.py", line 451, in process_maybe_execute_sell
     result = self.handle_stoploss_on_exchange(trade)
   File "/freqtrade/freqtrade/freqtradebot.py", line 625, in handle_stoploss_on_exchange
     if trade.stoploss_order_id else None
   File "/freqtrade/freqtrade/exchange/exchange.py", line 52, in wrapper
     return f(*args, **kwargs)
   File "/freqtrade/freqtrade/exchange/exchange.py", line 680, in get_order
     order = self._dry_run_open_orders[order_id]
 KeyError: 'dry_run_sell_753815'
```

database:

```
"8"	"8"	"binance"	"GO/BTC"	"1"	"0.001"	"0.001"	"1.27e-06"	"1.27e-06"				"0.005"	"3937.00787401575"	"2019-08-27 18:37:34.207821"			"1.33575e-06"	"-0.025"	"1.2065e-06"	"-0.05"	"dry_run_sell_753815"	"2019-08-27 18:41:46.859039"	"1.37e-06"	"1.27e-06"		"SignalStrategy"	"1"
```

### Steps to reproduce:

  1. run freqtrade with this configuration

```
{
  "max_open_trades": 5,
  "stake_currency": "BTC",
  "stake_amount": 0.005,
  "fiat_display_currency": "USD",
  "dry_run_wallet": 1,
  "dry_run": true,
  "db_url": "sqlite:////user_data/backtest_data/tradesv3.sqlite",
  "strategy": "SignalStrategy",
  "strategy_path": "/user_data/strategies",
  "unfilledtimeout": {
    "buy": 10,
    "sell": 30
  },
  "bid_strategy": {
    "ask_last_balance": 0.0,
    "use_order_book": true,
    "order_book_top": 1,
    "check_depth_of_market": {
      "enabled": true,
      "bids_to_ask_delta": 1
    }
  },
  "order_time_in_force": {
    "buy": "fok",
    "sell": "gtc"
  },
  "ask_strategy": {
    "use_order_book": true,
    "order_book_min": 1,
    "order_book_max": 9
  },
  "exchange": {
    "name": "binance",
    "key": "[...]",
    "secret": "[...]",
    "ccxt_config": {
      "enableRateLimit": true
    },
    "ccxt_async_config": {
      "enableRateLimit": true,
      "rateLimit": 900
    },
    "pair_whitelist": [
      "ADA/BTC",
      "ADX/BTC",
      "AE/BTC",
      [ ... ]
    ],
    "pair_blacklist": [
      "XMR/BTC"
    ],
    "_ft_has_params": {
      "ohlcv_candle_limit": 200,
      "ohlcv_partial_candle": false
    }
  },
  "edge": {
    "enabled": false,
    "process_throttle_secs": 3600,
    "calculate_since_number_of_days": 60,
    "capital_available_percentage": 0.5,
    "allowed_risk": 0.01,
    "stoploss_range_min": -0.01,
    "stoploss_range_max": -0.1,
    "stoploss_range_step": -0.01,
    "minimum_winrate": 0.60,
    "minimum_expectancy": 0.20,
    "min_trade_number": 10,
    "max_trade_duration_minute": 1440,
    "remove_pumps": false
  },
  "telegram": {
    "enabled": true,
    "token": "[...]",
    "chat_id": "[...]"
  },
  "initial_state": "running",
  "forcebuy_enable": true,
  "internals": {
    "process_throttle_secs": 5
  },
  "order_types": {
    "buy": "market",
    "sell": "limit",
    "stoploss": "limit",
    "stoploss_on_exchange": true,
    "stoploss_on_exchange_interval": 60
  },
  "api_server": {
    "enabled": true,
    "listen_ip_address": "0.0.0.0",
    "listen_port": 8080,
    "username": "null",
    "password": "null"
  }
}
```

order created with forcebuy rest RPC

**Comment by il-katta** [comment]
another similar error with another conf. Order still created by RPC.
```
 2019-08-27 21:26:59,628 - freqtrade - ERROR - Fatal exception!
 Traceback (most recent call last):
   File "/freqtrade/freqtrade/main.py", line 49, in main
     worker.run()
   File "/freqtrade/freqtrade/worker.py", line 72, in run
     state = self._worker(old_state=state)
   File "/freqtrade/freqtrade/worker.py", line 110, in _worker
     self._throttle(func=self._process, min_secs=throttle_secs)
   File "/freqtrade/freqtrade/worker.py", line 123, in _throttle
     result = func(*args, **kwargs)
   File "/freqtrade/freqtrade/worker.py", line 134, in _process
     state_changed = self.freqtrade.process()
   File "/freqtrade/freqtrade/freqtradebot.py", line 135, in process
     state_changed |= self.process_maybe_execute_sell(trade)
   File "/freqtrade/freqtrade/freqtradebot.py", line 451, in process_maybe_execute_sell
     result = self.handle_stoploss_on_exchange(trade)
   File "/freqtrade/freqtrade/freqtradebot.py", line 683, in handle_stoploss_on_exchange
     self.handle_trailing_stoploss_on_exchange(trade, stoploss_order)
   File "/freqtrade/freqtrade/freqtradebot.py", line 696, in handle_trailing_stoploss_on_exchange
     if trade.stop_loss > float(order['info']['stopPrice']):
 KeyError: 'stopPrice'
```
bot still crash.

**Comment by il-katta** [comment]
dump of variable `order` before `KeyError`:
```
<class 'dict'>: {'id': 'dry_run_sell_426994', 'pair': 'MDA/ETH', 'price': 0.003493, 'amount': 27.19756309834639, 'cost': 0.09500108790252394, 'type': 'stop_loss_limit', 'side': 'sell', 'remaining': 27.19756309834639, 'datetime': '2019-08-27T21:06:12.757811+00:00', 'status': 'open', 'fee': None, 'info': {}}
```

**Comment by xmatthias** [comment]
This one is solved in the develop-branch too #2119.

You can disable stoploss on exchange when using dry-run (that's what the above PR does, too) - since the order cannot be on exchange when using dry-run.

will try to get a release out ASAP.
sorry for the inconvenience.

**Comment by xmatthias** [comment]
Please update to version 2019.8 (released just now).

That will fix this issue.
